cli/README.md

```diff
@@ -46,14 +46,15 @@ To change the output format of the session recording use option
 `--sessionOutput log` 
 with start, stop, or call in between starting and stopping.
 Valid output options are:
+
 - log (default): method call request and response objects ordered by the timestamp of the objects.
 - raw: list of method call objects containing both request and response in one object.
-- mock-overrides: a directory of method calls extrapolated from the raw format. This option converts the raw formats into json files or yaml files. A json file would be the response from a method call that took place when mock-firebolt processed it. Yaml files contain a function that returns a json response depending on input params. Yaml files are only generated if there were multiple of the same method call with different params. 
+- mock-overrides: a directory of method calls extrapolated from the raw format. This option converts the raw formats into json files or yaml files. A json file would be the response from a method call that took place when mock-firebolt processed it. Yaml files contain a function that returns a json response depending on input params. Yaml files are only generated if there were multiple of the same method call with different params.
+- live: This format operates similarly to the 'log' format with an added real-time feature. As each message is received, it gets immediately written to the specified output file. In addition to accepting regular file paths, the 'live' option also supports WebSocket (WS/WSS) URLs. If a WS/WSS URL is designated as the outputPath, a WebSocket connection is established with the specified URL, and the new messages are dispatched to that connection. Please note that specifying an outputPath is essential for the 'live' option. This path is necessary whether you're sending the live log to a WebSocket URL or saving a live copy of the log file to a local directory.
```

Comment:
To confirm it looks like rather than imitate "raw" we're doing both the logic of "log" and the "live output" logic?

When we "stop()" a session do we get two files? The live output and the "log" output?

Reply:
Question 1: Sorry I have provided incorrect information in the readme. The "live" functionality does indeed imitate "raw" and not "log". I will correct the documentation.

Question 2: When we call stop() we will indeed have two files. One "live" file and another "log" file. Do we want to modify the logic to only provide the "live" file? It is a simple, one-line code change to do so.

Reply:
Updated documentation